### ISSUE TITLE
Importing toyboxes breaks in 2.0.1

### ISSUE THREAD

**Issue by GammaGames** [DidierMalenfant/toybox.py#10]
Ran into this, worked fine <2.0.0 but 2.0.1 breaks the `toyboxes.lua` file.

### Old:
```lua
import 'github-dot-com/Whitebrim/AnimatedSprite/AnimatedSprite.lua'
```

### New:
```lua
import '../toyboxes/github-dot-com/Whitebrim/AnimatedSprite/AnimatedSprite.lua'
```

For some reason it's all relative to the `source` folder?

**Comment by GammaGames** [comment]
If this is unexpected (I think it is) we can report it to the forums, neven recommended that on the discord

**Comment by DidierMalenfant** [comment]
Yeah this is `pdc` breaking relative includes.... again...

**Comment by DidierMalenfant** [comment]
opened a [thread](https://devforum.play.date/t/sdk-2-0-1-breaks-relative-path-imports-again/12009) on the dev forums.

**Comment by DidierMalenfant** [comment]
No worries. It's got to be a bug. That makes no sense to not have import be relative to the folder a file is in.

**Comment by tjkirch** [comment]
Thanks for jumping on this!

On the plus side, after 2.0.1, this note on the README should no longer be required, right?  `Note that due to a bug in the pdc app used by the Playdate SDK to process source files, the .lua extension is required here. Once this bug is fixed, this will no longer be needed.`  The 2.0.1 changelog said `pdc: fixed bug requiring lua file extensions when import path contains a period`.  (I'm new to toybox, so I'm not sure this is strictly related to the issue above.)

**Comment by DidierMalenfant** [comment]
> On the plus side, after 2.0.1, this note on the README should no longer be required, right? `Note that due to a bug in the pdc app used by the Playdate SDK to process source files, the .lua extension is required here. Once this bug is fixed, this will no longer be needed.`

Yep. I just tested that and they did fix it. I'll post and update when all this is resolved.

I think you're right and the relative path breaking has to do with a regression bug when they fixed other import things on the list.

**Comment by rubybrowncoat** [comment]
Is there anything we can do in the meantime to import toyboxes?

**Comment by GammaGames** [comment]
I have a solution in the original ticket, you have to do it whenever you update with toybox

**Comment by DidierMalenfant** [comment]
Maybe I can put a temp cmdline option that will do that for you in the meantime?

**Comment by rubybrowncoat** [comment]
> I have a solution in the original ticket, you have to do it whenever you update with toybox

Mmm I have done that but I'm having an issue even getting to toyboxes.lua, maybe I'm having a slightly different issue then? I'm getting this error on pdc:
`error: v:\Project\source/main.lua:5: No such file: ../toyboxes/toyboxes.lua`

(The command is: `"~\PlaydateSDK\bin\pdc" -sdkpath "~\PlaydateSDK" "v:\Project\source" "v:\Project\Windbreaker.pdx"`)

**Comment by DidierMalenfant** [comment]
That sounds like maybe something unrelated... I could take a look if the project is shareable.

**Comment by GammaGames** [comment]
It's (allegedly) fixed in 2.0.3! Any confirmations? I can test tonight

**Comment by DidierMalenfant** [comment]
Seems fixed. Let me know if it is for you too and I'll close this.

**Comment by GammaGames** [comment]
Confirming, I was able to `toybox -f update` and the project built without issue.
